packages/yoroi-extension/app/components/wallet/send/WalletSendFromSteps/#WalletSendPreviewStep.js#

```diff
@@ -0,0 +1,391 @@
+  +receivers: Array<string>,
+  +totalAmount: MultiToken,
+  +transactionFee: MultiToken,
+  +transactionSize: ?string,
+  +onSubmit: ({| password: string |}) => PossiblyAsync<void>,
+  +addressToDisplayString: string => string,
+  +onCancel: void => void,
+  +isSubmitting: boolean,
+  +error: ?LocalizableError,
+  +classicTheme: boolean,
+  +unitOfAccountSetting: UnitOfAccountSettingType,
+  +getTokenInfo: $ReadOnly<Inexact<TokenLookupKey>> => $ReadOnly<TokenRow>,
+  +getCurrentPrice: (from: string, to: string) => ?number,
+|};
+
+const messages = defineMessages({
+  nAssets: {
+    id: 'wallet.send.form.preview.nAssets',
+    defaultMessage: '!!!{number} Assets',
+  }
+});
+
+@observer
+export default class WalletSendPreviewStep extends Component<Props> {
+
+  static contextTypes: {|intl: $npm$ReactIntl$IntlFormat|} = {
+    intl: intlShape.isRequired,
+  };
+
+  form: ReactToolboxMobxForm = new ReactToolboxMobxForm({
+    fields: {
+      walletPassword: {
+        type: 'password',
+        label: this.context.intl.formatMessage(globalMessages.walletPasswordLabel),
+        placeholder: this.props.classicTheme ?
+          this.context.intl.formatMessage(globalMessages.walletPasswordFieldPlaceholder) : '',
+        value: '',
+        validators: [({ field }) => {
+          if (field.value === '') {
+            return [false, this.context.intl.formatMessage(globalMessages.fieldIsRequired)];
+          }
+          return [true];
+        }],
+      },
+    }
+  }, {
+    options: {
+      validateOnChange: true,
+      validationDebounceWait: config.forms.FORM_VALIDATION_DEBOUNCE_WAIT,
+    },
+    plugins: {
+      vjf: vjf()
+    },
+  });
+
+  submit(): void {
+    this.form.submit({
+      onSuccess: async (form) => {
+        const { walletPassword } = form.values();
+        const transactionData = {
+          password: walletPassword,
+        };
+        await this.props.onSubmit(transactionData);
+      },
+      onError: () => {}
+    });
+  }
+
+  convertedToUnitOfAccount: (TokenEntry, string) => string = (token, toCurrency) => {
+    const tokenInfo = this.props.getTokenInfo(token);
+
+    const shiftedAmount = token.amount
+      .shiftedBy(-tokenInfo.Metadata.numberOfDecimals);
+
+    const ticker = tokenInfo.Metadata.ticker;
+    if (ticker == null) {
+      throw new Error('unexpected main token type');
+    }
+    const coinPrice = this.props.getCurrentPrice(tick
+      tokenInfo.Metadata.ticker,
+      toCurrency
+    );
+
+    if (coinPrice == null) return '-';
+
+    return calculateAndFormatValue(
+      shiftedAmount,
+      coinPrice
+    );
+  }
+
+  renderSingleAmount: TokenEntry => Node = (entry) => {
+    const formatValue = genFormatTokenAmount(this.props.getTokenInfo);
+
+    const { unitOfAccountSetting } = this.props;
+    return unitOfAccountSetting.enabled
+      ? (
+        <>
+          <div className={styles.amount}>
+            {this.convertedToUnitOfAccount(entry, unitOfAccountSetting.currency)}
```

Comment:
Please fix the misalignment  
<img width="474" alt="Screen Shot 2022-06-03 at 12 12 48 PM" src="https://user-images.githubusercontent.com/72753578/171835281-08307a5a-569a-4132-a034-efc62c186bb1.png">

Reply:
I fixed it in this [PR](https://github.com/Emurgo/yoroi-frontend/pull/2832)